src/main/java/seedu/address/model/person/Parent.java

```diff
@@ -0,0 +1,88 @@
+package seedu.address.model.person;
+
+import java.util.Objects;
+import java.util.Set;
+
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.model.tag.Tag;
+
+/**
+ * Represents a parent in the address book
+ */
+public class Parent extends Person {
+    private Student child = null;
+
+    /**
+     * Constructs a {@code Parent} with the given details.
+     */
+    public Parent(Name name, Phone phone, Email email, Address address, Student child, Set<Tag> tags) {
+        super(name, phone, email, address, tags);
```

Comment:
Perhaps more tests could be written for this to hit minimum code coverage as determined by codecov!